src/ORM/DataQuery.php

```diff
@@ -1142,16 +1203,51 @@ public function addSelectFromTable($table, $fields)
 
     /**
      * Query the given field column from the database and return as an array.
+     * querying DB columns of related tables is supported but you need to make sure that the related table
+     * is already available in join
+     *
+     * @see DataList::applyRelation()
+     *
+     * example use:
+     *
+     * <code>
+     *  column("<table_name>"."<column_name>")
+     * </code>
      *
      * @param string $field See {@link expressionForField()}.
      * @return array List of column values for the specified column
+     * @throws InvalidArgumentException
```

Comment:
This could be considered a breaking change. Currently this method does already throw an InvalidArgumentException via `DataObjectSchema::sqlColumnForField()`, but perhaps not in the same context as this - although the logic does look fairly similar. Would you mind confirming?